lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1077,22 +1087,39 @@ public void unpublishObjectsExternally(Set<GUID> guids, int accessGroupId) throw
         for (Map<String, Object> hit : hitList) {
             @SuppressWarnings("unchecked")
             Map<String, Object> obj = (Map<String, Object>) hit.get("_source");
-            ObjectData item = buildObjectData(obj, pp.objectInfo, pp.objectKeys, 
-                    pp.objectData, pp.objectDataIncludes);
+            Map<String, ArrayList> highlightRes = (Map<String, ArrayList>) hit.get("highlight");
+            ObjectData item = buildObjectData(obj, editHighlightedKeys(highlightRes), pp);
             ret.add(item);
         }
         return ret;
     }
 
+    private Map<String, ArrayList> editHighlightedKeys(Map<String, ArrayList> highlightRes){
+        if(highlightRes == null){
+            return null;
+        }
+        Map<String, ArrayList> highlightResEdited = new HashMap<>();
+        for(String key : highlightRes.keySet()){
```

Comment:
spaces

Reply:
not going to make any more spaces comments but there's more in this PR

Reply:
I cant guarantee I can catch all of them. Feels like Looking for air bubbles in a frozen lake.

Reply:
NOT ACCEPTABLE

Reply:
or maybe if there are enough thumbs sticking out it will look flat